app/src/lib/stores/app-store.ts

```diff
@@ -327,9 +327,18 @@ export class AppStore extends TypedBaseStore<IAppState> {
    */
   private appIsFocused: boolean = false
 
-  private sidebarWidth: number = defaultSidebarWidth
-  private commitSummaryWidth: number = defaultCommitSummaryWidth
-  private stashedFilesWidth: number = defaultStashedFilesWidth
+  private sidebarWidth: number = getNumber(
```

Comment:
We already update these values in `loadInitialState()`, and I'm curious whether that isn't necessary any more (tradeoffs aside):

https://github.com/desktop/desktop/blob/be399eb44f3a03ed67fc9ae8b52affdaa4c82f7a/app/src/lib/stores/app-store.ts#L1570-L1574

Alternatively, could we just update `stashedFileChanges` there?

Reply:
yup, just saw that. totally my mistake. updating now

Reply:
🤦‍♀ f5afa9d